edg/parts/Lcd_Er_Tft1_28_3.py

```diff
@@ -0,0 +1,105 @@
+from edg.parts.PassiveConnector_Fpc import Fpc050BottomFlip
+from edg import *
+from edg.abstract_parts import *
+
+
+class Er_Tft_128_3_Outline(InternalSubcircuit, FootprintBlock):
+    """Footprint for TFT panel outline"""
+
+    def contents(self) -> None:
+        super().contents()
+        self.footprint('U', 'edg:Lcd_Er_Tft1_28_3_Outline', {},
+                       'EastRising', 'ER-TFT1.28-3',
+                       datasheet='https://www.buydisplay.com/download/manual/ER-TFT1.28-3_Datasheet.pdf')
+
+
+class Er_Tft_128_3_Device(InternalSubcircuit, Nonstrict3v3Compatible, Block):
+    """15-pin FPC connector for the ER-TFT1.28-3 device"""
+
+    def __init__(self) -> None:
+        super().__init__()
+
+        self.conn = self.Block(Fpc050BottomFlip(length=15))
```

Comment:
can I flip the pin number order and add comments about it, or did you mean to create a new footprint for it?

Reply:
Yes, I would recommend flipping the pin order in the HDL (plus comment for why it doesn't match the datasheet) and using the stock footprint. I prefer keeping footprints stock since they sometimes get updated upstream.

Reply:
Okay, the current version now flips the order in HDL